airflow/providers/http/hooks/http.py

```diff
@@ -320,6 +320,7 @@ async def run(
         data: dict[str, Any] | str | None = None,
         headers: dict[str, Any] | None = None,
         extra_options: dict[str, Any] | None = None,
+        data_type: Literal["json", "dict"] = "json",
```

Comment:
hmm.... 
why don't you just have params `json` and `data` and use whichever one you get?

Reply:
yeah i think your first attempt was the best move: 
https://github.com/apache/airflow/pull/37293/commits/01fee3ef57b28b6c5cbe551e982e04278df7a7ef

it was just a bug. json should go to json, and data to data, no?

Reply:
I like the idea of having two parameters one for data and other for json, let me try it.